mavros_extras/src/plugins/odom.cpp

```diff
@@ -126,6 +126,10 @@ class OdometryPlugin : public plugin::PluginBase {
 		auto cov_ang_vel = ftf::transform_frame_aircraft_baselink(odom->twist.covariance);
 		ftf::covariance_to_mavlink(cov_ang_vel, att.covariance);
 
+		ROS_INFO("position cov: \n%1.20f \n%1.20f \n%1.20f \n%1.20f \n%1.20f \n%1.20f \n%1.20f",
+		cov_pos_linvel[0], cov_pos_linvel[1], cov_pos_linvel[7],  cov_pos_linvel[14],
+		cov_pos_linvel[21], cov_pos_linvel[28],  cov_pos_linvel[35]);
```

Comment:
Why not something like that:

```cpp
ftf::EigenMapCovariance6d map(cov_pos_linvel.data());

ROS_INFO_STREAM("pos cov: " << std::endl << map);
```

And i hope that only for debugging?
Eigen::Map should work like Eigen::Matrix.